I2CDEVICES.md

```diff
@@ -66,3 +66,4 @@ Index | Define              | Driver  | Device   | Address(es) | Description
   42  | USE_DS1624          | xsns_59 | DS1624   | 0x48 - 0x4F | Temperature sensor
   43  | USE_AHT1x           | xsns_63 | AHT10/15 | 0x38        | Temperature and humidity sensor
   44  | USE_WEMOS_MOTOR_V1  | xdrv_34 |          | 0x2D - 0x30 | WEMOS motor shield v1.0.0 (6612FNG)
+  92  | USE_HDC1080         | xsns_92 | HDC1080  | 0x40        | Temperature and Humidity sensor
```

Comment:
Keep incrementing the Index as it has no relation to file name but only to startup sequenc. So change the index to 45.

Reply:
@arendst got it.